### ISSUE TITLE
CEMU broken after windows 24H2 update

### ISSUE THREAD

**Issue by lleandrhus** [cemu-project/Cemu#1434]
### Current Behavior

Can anyone tell me why it closes when it's loading Vulkan pipelines. I'm from Brazilian i don't speak English

### Expected Behavior

After loading the shaders the game starts.

### Steps to Reproduce

I don't know

### System Info (Optional)

OS: 
GPU:

**Comment by Squall-Leonhart** [comment]
The issues tracker is not the location for someone to tell you why, it is for reporting bugs by including as much information to reproducing the issue as possible.

Cemu is tested and runs fine on 24h2, and i can only assume you have a Radeon gpu given you did not include that information, which has a known driver regression that causes this crash behavior.

**Comment by lleandrhus** [comment]
I wonder if I posted in the wrong place and if I didn't put as much information as possible. I'm just a regular user and since windows updated Cemu stopped working and yes my card is a Radeon RX 6600.

**Comment by dogoawesomeness256** [comment]
yeah i have 6700 XT and im unable to play BOTW on the emulator other games like bayonetta run though
